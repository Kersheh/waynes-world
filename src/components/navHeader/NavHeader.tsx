import React from 'react';

import styles from './NavHeader.module.scss';

const NavHeader = () => {
  return <div className={styles.navHeader} />;
};

export default NavHeader;
