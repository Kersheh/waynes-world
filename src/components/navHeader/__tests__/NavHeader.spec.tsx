import React from 'react';
import { render } from '@testing-library/react';

import NavHeader from '../NavHeader';

describe('<NavHeader/> component', () => {
  it('should render', () => {
    const { container } = render(
      <NavHeader />
    );

    expect(container).toMatchSnapshot();
  });
});
