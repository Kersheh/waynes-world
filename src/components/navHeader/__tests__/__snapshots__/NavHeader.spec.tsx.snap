// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavHeader/> component should render 1`] = `
<div>
  <div
    class="navHeader"
  />
</div>
`;
